VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: BCM Instrumentcluster Gateway


BO_ 256 BCM_Status: 8 BCM
 SG_ Door_FL : 0|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway
 SG_ Door_FR : 1|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway
 SG_ Door_RL : 2|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway
 SG_ Door_RR : 3|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway

BO_ 513 BCM_Light_Status: 8 BCM
 SG_ TurnLeft : 0|1@1+ (1,0) [0|1] ""  Gateway,Instrumentcluster
 SG_ TurnRight : 1|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway
 SG_ Hazard : 2|1@1+ (1,0) [0|1] ""  Instrumentcluster,Gateway



BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "BusType" "CAN";

